packages/card/src/react/index.tsx

```diff
@@ -0,0 +1,359 @@
+import polyfillFocusWithin from 'focus-within'
+
+import { sizes as iconSizes } from '@pluralsight/ps-design-system-icon'
+import {
+  names as themeNames,
+  useTheme
+} from '@pluralsight/ps-design-system-theme'
+import {
+  canUseDOM,
+  omit,
+  RefForwardingComponent,
+  ValueOf
+} from '@pluralsight/ps-design-system-util'
+import { compose, css, StyleAttribute } from 'glamor'
+import * as React from 'react'
```

Comment:
```suggestion
import React from 'react'
```

to match the more common pattern we use

Reply:
Not sure why we wouldn't keep the new pattern. We're going to go in that direction eventually. I'm not stressed about going there piecemeal instead of en masse at a future point.  You ok with piecemeal, per-package?

Reply:
are you saying `* as React` is the new pattern. i don't really know what is the correct/current approach

with React 17 we don't need to import React at all for jsx...so we'd just pull in named exports? maybe? not sure

Reply:
Yes, I understand `* as React` to be the eventually-only supported import pattern.  Currently, you can go either way. 

See:
https://twitter.com/sebmarkbage/status/1250285076014747648

React 17 removes the need to use the react import as the jsx pragma.  But, we'll still need it for things like hooks or typings.